components/eventing-controller/controllers/subscriptionv2/jetstream/test_utils_test.go

```diff
@@ -558,7 +559,7 @@ func StartAndWaitForWebhookServer(k8sManager manager.Manager, webhookInstallOpts
 	dialer := &net.Dialer{Timeout: time.Second}
 	addrPort := fmt.Sprintf("%s:%d", webhookInstallOpts.LocalServingHost, webhookInstallOpts.LocalServingPort)
 	err := retry.Do(func() error {
-		//nolint:gosec //the test certificate used will report as bad certificate and hence not perform the test
+		//nolint:nolintlint //the test certificate used will report as bad certificate and hence not perform the test
```

Comment:
https://imgflip.com/i/780ghu

Reply:
Well, in the end it passed our pipeline, "and nothing else matters"